rq/job.py

```diff
@@ -750,4 +748,58 @@ def register_dependency(self, pipeline=None):
             connection.sadd(dependents_key, self.id)
             connection.sadd(self.dependencies_key, dependency_id)
 
+    @property
+    def dependencies_job_ids(self):
+        dependencies = self.connection.smembers(self.dependencies_key)
+        return [Job.key_for(as_text(_id))
```

Comment:
I think we can drop `as_text`, since Python 2.7 has been discontinued. Let's new codes Py 3 only from now on.

Reply:
0672cd00c6ab5268f83887e66990726904cbe1f9